res/controllers/Hercules DJ Control MP3 e2.midi.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<MixxxMIDIPreset schemaVersion="1" mixxxVersion="1.8.0+">
+<MixxxMIDIPreset schemaVersion="1" mixxxVersion="1.12.0+">
   <info>
     <name>Hercules DJ Control MP3 e2</name>
     <author>Gianfe</author>
```

Comment:
is it possible to add and not replace ?
like <name>Author1, Author2</name>

It would be better as I used large part of previous work.